### PR TITLE
Auto update version number on home page

### LINKED ISSUE #192: Automatically update the version number on the home page
When a new version of FuncUnit is released and the website is updated, the home page should have the updated version number.

Related to https://github.com/bitovi/funcunit/pull/180

### PR DESCRIPTION
**Pull request by Rlin415**
Definitely need a review. This closes #192

### CODE REVIEW

**Comment by chasenlehara** [comment]
Hi @Rlin415, please make sure you write [good commit messages](https://chris.beams.io/posts/git-commit/) that include a reference to the original issue and an explanation of the change. In general, you should be able to use your commit message(s) as a basis for providing a thorough PR description of the change you’re proposing.

You might consider [amending the commit message](https://nathanhoad.net/git-amend-your-last-commit) and [force pushing](http://movingfast.io/articles/git-force-pushing/) this branch.

File: index.html [hunk 1]
```diff
@@ -73,20 +73,14 @@
...
 	<div class="hero">
 		<div class="container">
 			<span class="hero-background iconNew-fu-logo-icon"></span>
 			<div id="hero-download" class="pull-right">
 				<h1>Write better tests, faster.</h1>
 
-				<a href="https://raw.githubusercontent.com/bitovi/funcunit/v3.1.0/dist/funcunit.js" download="funcunit.js">
-					<button class="download">Download 3.1.0<span>(One file. Everything you need.)</span></button>
+				<a href="https://raw.githubusercontent.com/bitovi/funcunit/v3.2.0/dist/funcunit.js" download="funcunit.js">
+					<button class="download">Download 3.2.0<span>(One file. Everything you need.)</span></button>
```
Comment (chasenlehara) [inline index.html]: @Rlin415, can you explain this PR a little bit better? This is still hard-coded, so it’s unclear whether the issue is actually fixed or not.

**Comment by Rlin415** [comment]
@chasenlehara Sure. So what I did was basically add a template and helper function named `getVersion` for the home page. This was necessary because now we can dynamically generate the html for the home page when we do `npm run document` which allows us to update the version number accordingly each time. The generated html file for the home page is called `index.html` and is placed in the root directory as well so I could see why you got confused. It looks like I just hard coded the version number in, but it was in fact the generated html file for the home page after running `npm run document`. I put the `index.html` file in the `.gitignore` but it wasn't ignored :(

File: documentjs.json [hunk 2]
```diff
@@ -15,10 +15,16 @@
 			},
 			"parent": "Guides",
 			"pageConfig": {"page":"guides"}
+		},
+		"funcunit": {
+			"glob": {
+				"pattern": "funcunit.md"
```
Comment (chasenlehara) [inline documentjs.json]: I don’t think this file should be necessary, so I would double check that any of this configuration is required.

**Pull request merged**